crates/re_analytics/src/lib.rs

```diff
@@ -54,76 +59,51 @@ pub enum EventKind {
 }
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
-pub struct Event {
+pub struct AnalyticsEvent {
     // NOTE: serialized in a human-readable format as we want end users to be able to inspect the
     // data we send out.
     #[serde(with = "::time::serde::rfc3339")]
-    pub time_utc: OffsetDateTime,
-    pub kind: EventKind,
-    pub name: Cow<'static, str>,
-    pub props: HashMap<Cow<'static, str>, Property>,
+    time_utc: OffsetDateTime,
+    kind: EventKind,
+    name: Cow<'static, str>,
+    props: HashMap<Cow<'static, str>, Property>,
 }
 
-impl Event {
-    pub fn append(name: impl Into<Cow<'static, str>>) -> Self {
+impl AnalyticsEvent {
+    #[inline]
+    pub fn new(name: impl Into<Cow<'static, str>>, kind: EventKind) -> Self {
         Self {
             time_utc: OffsetDateTime::now_utc(),
-            kind: EventKind::Append,
+            kind,
             name: name.into(),
             props: Default::default(),
         }
     }
 
-    pub fn update(name: impl Into<Cow<'static, str>>) -> Self {
-        Self {
-            time_utc: OffsetDateTime::now_utc(),
-            kind: EventKind::Update,
-            name: name.into(),
-            props: Default::default(),
-        }
+    #[inline]
+    pub fn append(name: impl Into<Cow<'static, str>>) -> Self {
```

Comment:
It would be nice with a docstring explaining what `append` is for, and how it compares to `insert`

Reply:
These are constructors, but they are no longer in use, so I removed them